crates/module-system/sov-modules-api/src/transaction/mod.rs

```diff
@@ -271,6 +419,24 @@ impl<R: TransactionCallable, S: Spec> Transaction<R, S> {
                     .verify(&inner.pub_key, msg, meter)
                     .map_err(TransactionVerificationError::from)?;
             }
+            VersionedTx::V1(inner) => {
+                for signature in inner.signatures.iter() {
+                    // Charge gas for the signature verification with out verifying
+                    MeteredSignature::new::<S>(signature.signature.clone())
```

Comment:
We charge for all signature verifications, even if the first one fails. This simplifies reasoning about the overall cost, but let's add a comment to make it clear that this behavior is intentional.